tools/tiled_ensemble/test_ensemble.py

```diff
@@ -0,0 +1,124 @@
+"""Anomalib Testing Script for ensemble of models.
```

Comment:
I guess this needs to be updated as well.

Reply:
I'll go over the test and rewrite that.